homeassistant/helpers/config_validation.py

```diff
@@ -552,6 +552,19 @@ def uuid4_hex(value):
     return result.hex
 
 
+def zwave_network_key(value):
+    """Validate a 16 byte value for zwave network keys."""
+    regex = re.compile(r'(0x\w\w,\s?){15}0x\w\w')
+    try:
+        if not regex.match(value):
+            raise vol.Invalid('Invalid Z-Wave network key')
+    except (ValueError, AttributeError, TypeError) as error:
```

Comment:
When would these errors happen? Could we change the order of validation to avoid them? Eg validate string first.